### DISCUSSION TITLE
Acount

### DISCUSSION THREAD

**Discussion by Eaielectronic** [PenguinMod/PenguinMod-Home#125] [Q&A]
Why when I connect with my scratch account it sends me to a canot get api user page…

**Comment by JeremyGamer13** [comment]
the servers are down
![image](https://github.com/user-attachments/assets/111a4cdf-0a7c-4977-8c9b-944503443584)

**Comment by Eaielectronic** [comment]
Yes, I saw it, but it's been like this for a few months, so when will it work again?

**Comment by derpygamer2142** [comment]
servers are currently in beta, hopefully soon
